src/electron/office/office_web_plugin.cc

```diff
@@ -278,8 +278,17 @@ blink::WebInputEventResult OfficeWebPlugin::HandleKeyEvent(
       return blink::WebInputEventResult::kNotHandled;
   }
 
-  // intercept some special key events on Ctr/Command
-  if (event.GetModifiers() & (event.kControlKey | event.kMetaKey)) {
+  blink::WebInputEvent::Modifiers base_modifier = event.kControlKey;
+
+  bool is_mac = false;
+
+#if BUILDFLAG(IS_MAC)
+  base_modifier = event.kMetaKey;
+  is_mac = true;
```

Comment:
Instead of `is_mac`, it's better to just use another `#if` block below, since the code will only ever run on mac.

Reply:
Ok, will do.